pkg/guacanalytics/patchPlanning.go

```diff
@@ -158,35 +171,49 @@ func exploreIsDependencyFromDepPkg(ctx context.Context, gqlclient graphql.Client
 		return nil
 	}
 
-	dfsNVersion, seenVersion := q.nodeMap[isDependency.Package.Namespaces[0].Names[0].Versions[0].Id]
-	dfsNName, seenName := q.nodeMap[isDependency.Package.Namespaces[0].Names[0].Id]
+	addNodeToQueue(q, q.now, q.nowNode.Depth+1, "packageVersion", false, nil, isDependency.Package.Namespaces[0].Names[0].Versions[0].Id)
+	addNodeToQueue(q, q.now, q.nowNode.Depth+1, "packageName", false, []string{isDependency.Package.Namespaces[0].Names[0].Versions[0].Version}, isDependency.Package.Namespaces[0].Names[0].Id)
 
-	if !seenVersion {
-		dfsNVersion = DfsNode{
-			Parent:   q.now,
-			Depth:    q.nowNode.Depth + 1,
-			NodeType: "packageVersion",
-		}
-		q.nodeMap[isDependency.Package.Namespaces[0].Names[0].Versions[0].Id] = dfsNVersion
-	}
+	return nil
+}
 
-	if !seenName {
-		dfsNName = DfsNode{
-			Parent:       q.now,
-			Depth:        q.nowNode.Depth + 1,
-			NodeType:     "packageName",
-			nodeVersions: []string{isDependency.Package.Namespaces[0].Names[0].Versions[0].Version},
+func exploreIsOccurrenceFromSubject(ctx context.Context, gqlclient graphql.Client, q *queueValues, isOccurrence model.NeighborsNeighborsIsOccurrence) {
+	addNodeToQueue(q, q.now, q.nowNode.Depth+1, "artifact", false, nil, isOccurrence.Artifact.Id)
+}
+
+func exploreHasSLSAFromArtifact(ctx context.Context, gqlclient graphql.Client, q *queueValues, hasSLSA model.NeighborsNeighborsHasSLSA) {
+	addNodeToQueue(q, q.now, q.nowNode.Depth+1, "artifact", true, nil, hasSLSA.Subject.Id)
```

Comment:
probably need to make sure that it is coming from the materials and not the subject here since neighbor doesn't distinguish if its the subject or the materials